staging/src/k8s.io/kube-scheduler/config/v1beta2/types.go

```diff
@@ -53,12 +53,6 @@ type KubeSchedulerConfiguration struct {
 	// ClientConnection specifies the kubeconfig file and client connection
 	// settings for the proxy server to use when communicating with the apiserver.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
-	// HealthzBindAddress is the IP address and port for the health check server to serve on,
-	// defaulting to 0.0.0.0:10251
-	HealthzBindAddress *string `json:"healthzBindAddress,omitempty"`
-	// MetricsBindAddress is the IP address and port for the metrics server to
-	// serve on, defaulting to 0.0.0.0:10251.
-	MetricsBindAddress *string `json:"metricsBindAddress,omitempty"`
```

Comment:
how is the address/port controlled now that these are removed?

Reply:
https://github.com/kubernetes/kubernetes/pull/96216#discussion_r582857638Set to the same address/port as the

Reply:
KS does the same. So both healthz/metrics honor the address/port.

Reply:
this was introduced this cycle, not yet released. The v1beta1 ones still exist and we validate similar to the port flag.

Reply:
treat this the same as v1beta1 now that v1beta2 is released (was released in 1.22)

Reply:
@liggitt can you more elaborate on what "the same" means here? I lost the context.

Reply:
v1beta2 is a released API. As such, we cannot remove fields from it, as it would be a breaking change. So we can only remove the field in v1beta3 or v1.

Reply:
Thanks!!! Addressed